config.py

```diff
@@ -13,6 +13,8 @@ def pad_angle_brackets(value):
     This issue occurs when a task order name contains something like:
     "Blah Blah project on lots <50 acres". It happens. meh. 
     """
+    if not value:
+        return value
```

Comment:
s3_to_knack.py was bombing here, because we were passing a `NoneType` for the value of `TASK_ORDER_DESC`. Should not happen anymore, but here's handling it just in case.